### DISCUSSION TITLE
Responsive images are not being generated

### DISCUSSION THREAD

**Discussion by ogradye** [spatie/laravel-medialibrary#2933] [Q&A]
`$car->addMediaFromDisk('/cars/tmp/' . $fileName, 'spaces')->withResponsiveImages()->toMediaCollection('cars');`

No responsive images are generated. When I called withResponsiveImages the following happens:
- the information is added to the media table with the responsive array empty. 
- The file is placed in storage/media-library/temp (I find this odd, and wouldn't like this to happen as the files should be stored on a remote object storage service)

No exceptions are thrown.

**Comment by Don404** [comment]
Is the image "webp" format?
